src/base/rss/rss_autodownloader.cpp

```diff
@@ -245,6 +245,7 @@ void AutoDownloader::processJob(const QSharedPointer<ProcessingJob> &job)
         }
 
         rule.setLastMatch(job->articleDate);
+        rule.matched(job->articleTitle);
```

Comment:
I've just been trying to implement this, but I came across the fact the matches method is const, and can't have side effects. That makes saving the computed episode name impossible. I think the correct solution is to have the matches method return an instance of a new object if it does match (which is what the regular expression class does), but I'm not sure changing the interface for the matches method is worth it for just the episode name.